appsso-starter-java/README.md

```diff
@@ -2,6 +2,35 @@
 
 This repository provides an example application used to set up an authentication mechanism with AppSSO.
 
-## Usage
+## Getting Started
 
+1. Discover your `AppSSO` service offerings
+
+   ```bash
+   tanzu services classes list
+   ```
+
+   If there isn't one,
+
+   - ask your service operator `OR`
+   - [create your own `ClusterUnsafeTestLogin`](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.6/tap/app-sso-reference-api-clusterunsafetestlogin.html)
+
+2. Select the service offering you'd like to connect
+
+4. Claim an instance from the offering
```

Comment:
**[issue]**: numbering in this list is skewed (1,2,4,5)

Reply:
Nice catch! Thanks